src/sentry/search/utils.py

```diff
@@ -171,14 +171,31 @@ def tokenize_query(query):
             results['query'].append(token)
             continue
 
+        # this handles quoted string, and is duplicated below
+        if token[0] == '"':
+            nvalue = token
+            while nvalue[-1] != '"':
```

Comment:
It's possible now that this can raise if len == 0. Not sure if that's possible here though, I haven't looked that closely. That's why I prposed `[-1:]`